docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+---
+services:
+  # run postgres for proxy service to store observability metrics
+  postgres:
+    image: postgres:15
+    env_file: .env
+    ports:
+      - "${POSTGRES_HOST_PORT}:${POSTGRES_CONTAINER_PORT}"
+  # run redis for proxy service to cache responses
+  redis:
+    image: 'bitnami/redis:latest'
```

Comment:
Why not the official redis image?

Reply:
I've always used bitnami images wherever possible, they do a better job of patching and updating their base images as they use them in production whereas the offical redis image is more caveat emptor